src/main/ts/components/Editor.ts

```diff
@@ -52,7 +52,7 @@ export const Editor = defineComponent({
         ...conf,
         readonly: props.disabled,
         selector: `#${elementId}`,
-        plugins: mergePlugins(conf.plugins, props.plugins),
+        plugins: mergePlugins(conf.plugins as string, props.plugins as string),
```

Comment:
These types are not right. They can allow string[] or they can be undefined. This is probably not a good place for a cast. Try updating the types on mergePlugins and normalisePluginArray
https://github.com/tinymce/tinymce-react/blob/main/src/main/ts/Utils.ts#L93
https://github.com/tinymce/tinymce-react/blob/main/src/main/ts/Utils.ts#L84

Reply:
- [ ] Fix types